### PR TITLE
Apply stylesheet changes and additions from newest SUL styles

### LINKED ISSUE #16: Some basic CSS settings
Some global settings from libweb:

```
body{font-size:15px;font-size:.9375rem;font-family:'Source Sans Pro', 'Arial Unicode MS', Helvetica, sans-serif;background-color:#fbfbf9;color:#2e2d29;line-height:1.6;overflow-x:hidden}

a{text-decoration:none}
a:hover{text-decoration:underline}
a{color:#a26624}

table{width:100%;border:0;border-collapse:collapse}
thead{text-align:left}
thead td{border:0}
td{padding:12px 12px 12px 0;border:0 solid #d5d0c0;border-width:1px 0 0;vertical-align:top}

h1{font-size:37px;font-size:2.3125rem}
@media (min-width: 48em){h1{font-size:45px;font-size:2.8125rem}}
h1.smallh1{font-size:37px;font-size:2.3125rem;font-weight:100;color:#5f574f}
h2{font-size:26px;font-size:1.625rem;font-weight:100;color:#8c1515}
@media (min-width: 48em){h2{font-size:31px;font-size:1.9375rem}}
h2.small,h2.small2{font-size:26px;font-size:1.625rem}
h2.sidebar{font-size:18px;font-size:1.125rem;font-weight:400;text-transform:uppercase}
h3,.intro{font-size:18px;font-size:1.125rem;color:#5f574f}
@media (min-width: 48em){h3,.intro{font-size:22px;font-size:1.375rem}}
h4{font-size:18px;font-size:1.125rem;font-weight:600;color:#5f574f}
h5{font-size:16.5px;font-size:1.03125rem;font-weight:600}
h6{font-size:15px;font-size:.9375rem;font-weight:600}
.tiny{font-size:13px;font-size:.8125rem}
```

### PR DESCRIPTION
**Pull request by jgreben**
Fixes #16

### CODE REVIEW

File: app/assets/stylesheets/base.scss [hunk 1]
```diff
@@ -10,14 +10,35 @@ a {
 
 h1 {
   color: $sul-h1-font-color;
-  font-size: 2.3em;
+  font-size: 2.1em;
   letter-spacing: -0.05em;
```
Comment (jvine) [inline app/assets/stylesheets/base.scss]: Kill letter-spacing - it's only relevant to SearchWorks when using deja-vu font.

File: app/assets/stylesheets/variables.scss [hunk 2]
```diff
@@ -11,7 +11,7 @@ $btn-default-border: $btn-beige-border;
 $headings-font-weight: 300;
 
 // Sizes
-$font-size-base: 14px;
+$font-size-base: 15px;
 $line-height-base: 1.428571429;
```
Comment (jvine) [inline app/assets/stylesheets/variables.scss]: change base line-height:1.6

File: app/assets/stylesheets/base.scss [hunk 3]
```diff
@@ -10,14 +10,35 @@ a {
...
 }
 
 h2 {
   color: $sul-h2-font-color;
-  font-size: 1.6em;
+  font-size: 1.3em;
+  font-weight: 100;
+}
+
+@media (min-width: 980px) {
```
Comment (jvine) [inline app/assets/stylesheets/base.scss]: I think these media settings are backwards. 

small media / default has this:
h1{font-size:37px;font-size:2.3125rem}  
h2{font-size:26px;font-size:1.625rem;font-weight:100;color:#8c1515}

large media has this:
@media (min-width: 48em){h1{font-size:45px;font-size:2.8125rem}}
@media (min-width: 48em){h2{font-size:31px;font-size:1.9375rem}}

**Review by jvine — COMMENTED** [review]
Not seeing the a settings - should they be in this section?
a{text-decoration:none}
a:hover{text-decoration:underline}
a{color:#a26624}

**Comment by jgreben** [comment]
@jvine the `a` settings were in the top-nav css and the form css. I removed the form.css because I don't think we need it (yet) and moved the `a` settings to base.css. I fixed the rest of the things you mentioned. 

Hers is the large media view:
large-media

Here is the small media view:
small-media

For now you'll have to take my word that the `h1` size gets smaller..

**Pull request merged**